components/input/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="inputstyle.css">
+</head>
+<body>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    
+    
+    <div class="input-box">
+        <input class="input1" placeholder="Email Id"></input>
+    </div>
+
+    <div class="input-icon">
+        <input class="input2" placeholder="Email Id"><i class="fas fa-user icon" aria-hidden="true"></i></input> </i>
```

Comment:
I'm not sure if you should the icon inside the input tag. it should be a sibling and then you can position it with absolute. looking at the out put it looks like this code works but not sure if this is the right thing to do.

Reply:
yeah, I have tried it but showing an error so I have tried this one.